schedule/supportserver_generator.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: create_hdd_gnome
+description: |
+  image creation job used as parent for other jobs testing based on existing installation.
+  To be used as `START_AFTER_TEST=create_hdd_gnome
+vars:
+  DESKTOP: gnome
+  HDDSIZEGB: '30'
+  INSTALLONLY: '1'
+  PUBLISH_HDD_1: SLES-%VERSION%-%ARCH%-Build209.2@%MACHINE%-gnome.qcow2
+  PUBLISH_PFLASH_VARS: SLES-%VERSION%-%ARCH%-Build209.2@%MACHINE%-gnome-uefi-vars.qcow2
+  SEPARATE_HOME: '0'
+schedule:
+  - installation/isosize
+  - installation/bootloader_uefi
```

Comment:
Just up to you, but you can use `bootloader_start` here. Once we'll plan to refactor our bootloader, it will be easier then to do so. Also, it will help in case we'll use the test on other than UEFI environment in future.